examples/regression.jl

```diff
@@ -0,0 +1,178 @@
+# A recreation of https://gpflow.readthedocs.io/en/master/notebooks/advanced/gps_for_big_data.html
```

Comment:
@rossviljoen what are the steps to render the notebook? I tried `Literate.notebook("examples/regression.jl"; execute=true)`, but the resulting .ipynb doesn't actually contain any plots. (I also had to add a couple dependencies - Optim and IterTools are used by the notebook but not in the SparseGPs.jl/Project.toml)

Reply:
They're not set up as notebooks or anything yet - they're basically just scripts at the moment. I was going to set up proper example notebooks with their own dependencies etc later.

Reply:
Ah, I was just hoping for recommended instructions to see a rendered version : )